P5/Source/Guidelines/en/WD-NonStandardCharacters.xml

```diff
@@ -1356,11 +1356,16 @@ The selection and combination of modules to form a TEI schema is described in
 <include xmlns="http://www.w3.org/2001/XInclude" href="../../Specs/charName.xml"/>
 <include xmlns="http://www.w3.org/2001/XInclude" href="../../Specs/charProp.xml"/>
 <include xmlns="http://www.w3.org/2001/XInclude" href="../../Specs/charDecl.xml"/>
+<!-- duncdrum: Why should / can I not include the att class here??  -->
+<!-- <include xmlns="http://www.w3.org/2001/XInclude" href="../../Specs/att.gaijiProp.xml"/> -->
```

Comment:
If I include this here i get two name and two value properties on the elements which leads odd validation errors (two name attribtes) and to display issues with the html spec page.